be/src/vec/sink/vmysql_result_writer.cpp

```diff
@@ -156,16 +161,24 @@ Status VMysqlResultWriter<is_binary_format>::append_block(Block& input_block) {
                                                                                            27);
                 }
             } else {
-                serde = block.get_by_position(i).type->get_serde();
+                serde = block.get_by_position(col_idx).type->get_serde();
             }
             serde->set_return_object_as_string(output_object_data());
             arguments.emplace_back(column_ptr.get(), col_const, serde);
         }
 
-        for (size_t row_idx = 0; row_idx != num_rows; ++row_idx) {
-            for (int i = 0; i < _output_vexpr_ctxs.size(); ++i) {
-                RETURN_IF_ERROR(arguments[i].serde->write_column_to_mysql(
-                        *(arguments[i].column), row_buffer, row_idx, arguments[i].is_const));
+        for (size_t row_idx = 0; row_idx < num_rows; ++row_idx) {
```

Comment:
Don’t check every row

Reply:
My fault.
Check loop is divided from for loop and placed in the front.